fv3gfs-physics/tests/savepoint/conftest.py

```diff
@@ -99,12 +112,32 @@ def read_serialized_data(serializer, savepoint, variable):
     return data
 
 
-def process_grid_savepoint(serializer, grid_savepoint, rank, *, backend: str):
-    grid = make_grid(grid_savepoint, serializer, rank, backend=backend)
-    fv3core._config.set_grid(grid)
+def upper_grid(serializer, grid_savepoint, rank, layout, *, backend: str):
+    grid = make_grid(grid_savepoint, serializer, rank, layout, backend=backend)
```

Comment:
Can you call `make_grid` directly?

I'm a little confused by this network of functions (`grid_data`, `grid_indexing`, `physics_config`, etc.). Instead of calling `grid_data(grid)`, should you directly use `grid.grid_data` in whatever function is calling these?

Reply:
They were supposed to be pytest fixtures, but it's easier to make them instead. I forgot to clean up, deleting all these now.